assemblies/assembly-configuring-the-global-header.adoc

```diff
@@ -0,0 +1,19 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: configuring-the-global-header-in-rhdh
+[id="{context}"]
+= Configuring the global header in {product}
+
+As an administrator, you can configure the {product} global header to create a consistent and flexible navigation bar across your {product-short} instance.
+By default, the {product} global header includes the following:
+
+* Search input field
+* Create button linking to software templates
+* Support for internal and external links, for example, support and notification icons
```

Comment:
I personally would not mention the notificaiton icon here since the next bullet point mention this explicit. Wdyt about:

```suggestion
* Support button that can link an internal or external support page
```